finalPrep/README.md

```diff
@@ -38,17 +38,17 @@ When you are finished, add screenshots of your calendar so we can provide feedba
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here:https://gist.github.com/BrendaSotel0/3b12d5b81c7abc348616283e1c4f55fd
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here:https://gist.github.com/BrendaSotel0/ab9b382ca3aa4cb871cd41eb3c1769ef
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist here: https://gist.github.com/BrendaSotel0/a2de7934efa8d3de790e86ac0e050eda
```

Comment:
Keep pondering these questions! You will have ample opportunity to do so during Turing. Just a note that I see you practice 'harsh honesty' and also like feedback 'as kind as possible'. I just highlight this because for me, Turing was a great practice in developing how I give and receive feedback. I always try to remember that feedback, however delivered, is a blessing and is showing that someone cares about how you are doing. Sometimes I would get defensive over feedback that seemed to be delivered harshly, but many times, it just came across that way because of time constraints, or because I was overly tired, frankly! Balancing brutal honesty with kindness is a skill that everyone will be working on at Turing.